src/main/java/com/redhat/sso/ninja/ManagementController.java

```diff
@@ -316,6 +309,23 @@ public Response databaseSave(@Context HttpServletRequest request,@Context HttpSe
     return Response.status(200).entity(Json.newObjectMapper(true).writeValueAsString(Database2.get())).build();
   }
 
+  // Admin UI to be able to update a single user field
+  @PUT
+  @Path("/users/{user}")
+  public Response updateUserProperty(@Context HttpServletRequest request, @PathParam("user") String user) throws JsonGenerationException, JsonMappingException, IOException{
```

Comment:
This causes duplicate methods to be found in the merged file

Reply:
Hey Andy - I'm not sure I'm quite following... are you saying you have duplicate methods in the ManagementController.java file when you checkout PR 129? I just checked out #129 and there was only one instance of the method called "updateUserProperty" in the ManagementController class -  do you get different results?

Reply:
I get this issue when attempting to merge in the changes in the branch with the master branch

Can you run the following with an updated master:

```
git checkout -b matallen-publish-events master
git pull https://github.com/matallen/ninja-board.git publish-events
```